oauth2client/client.py

```diff
@@ -836,13 +837,13 @@ def _do_revoke(self, http, token):
         query_params = {'token': token}
         token_revoke_uri = _helpers.update_query_params(
             self.revoke_uri, query_params)
-        resp, content = transport.request(http, token_revoke_uri)
-        if resp.status == http_client.OK:
+        response = transport._request(http, token_revoke_uri)
+        if response.status == http_client.OK:
             self.invalid = True
         else:
-            error_msg = 'Invalid response {0}.'.format(resp.status)
+            error_msg = 'Invalid response {0}.'.format(response.status)
             try:
-                d = json.loads(_helpers._from_bytes(content))
+                d = json.loads(_helpers._from_bytes(response.data))
```

Comment:
Since you dropped by, might as well give it a better name than `d`. `error_dict`?